contracts/facade/FacadeAct.sol

```diff
@@ -79,7 +79,9 @@ contract FacadeAct is IFacadeAct {
                         );
                     }
                 }
-            } else if (!cache.bh.fullyCollateralized()) {
+            } else if (
```

Comment:
Actually this condition should be status == CollateralStatus.SOUND. Sorry for not catching that on the first pass through. BackingManager.manageTokens reverts when IFFY

Reply:
So the check should be `if (cache.bh.status() == CollateralStatus.SOUND && !cache.bh.fullyCollateralized()) {` Is it the same for the call below? in Line 155?